### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by modridi**
Made a few changes to the Dockerfile
- fixed gopass & summon's installation
- repo:tag are now passed as ARG

Using a script for the moment to automate the build, not done yet (can be tested though..)

Backed up the old Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,41 +1,45 @@
...
+    SUMMON_VERSION=0.8.3 \
+    TERRAFORM_PROVISIONER_ANSIBLE_VERSION=2.5.0
 
 RUN apk add jq gnupg bash ansible
 
 # Install gopass
-RUN wget https://github.com/gopasspw/gopass/releases/download/v${GOPASS_VERSION}/gopass-${GOPASS_VERSION}-linux-amd64.tar.gz -qO - | \
-	tar xz gopass-${GOPASS_VERSION}-linux-amd64/gopass -O > /usr/local/bin/gopass
+RUN wget https://github.com/gopasspw/gopass/releases/download/v${GOPASS_VERSION}/gopass-${GOPASS_VERSION}-linux-amd64.tar.gz
+RUN tar -zxvf gopass-${GOPASS_VERSION}-linux-amd64.tar.gz gopass-${GOPASS_VERSION}-linux-amd64/gopass -O > /usr/local/bin/gopass
```
Comment (mcanevet) [inline Dockerfile:26]: Doing this will create an extra layer in the image so the image will be bigger than needed

File: Dockerfile [hunk 2]
```diff
@@ -1,41 +1,45 @@
...
 RUN chmod +x /usr/local/bin/gopass
+RUN rm gopass-${GOPASS_VERSION}-linux-amd64.tar.gz
 
 # Install summon
-RUN wget https://github.com/cyberark/summon/releases/download/v${SUMMON_VERSION}/summon-linux-amd64.tar.gz -qO - | \
-	tar xz summon -O > /usr/local/bin/summon
+RUN wget https://github.com/cyberark/summon/releases/download/v${SUMMON_VERSION}/summon-linux-amd64.tar.gz
+RUN tar -zxvf summon-linux-amd64.tar.gz summon -O > /usr/local/bin/summon
```
Comment (mcanevet) [inline Dockerfile:32]: same here

**Review by mcanevet — CHANGES_REQUESTED** [review]

File: Dockerfile.backup [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+# Install Terraform Ansible provisioner
+RUN mkdir -p x /tmp/.terraform.d/plugins \
+        && chgrp 0 -R /tmp/.terraform.d \
+        && chmod g+w -R /tmp/.terraform.d \
+        && wget -q https://github.com/radekg/terraform-provisioner-ansible/releases/download/v${TERRAFORM_PROVISIONER_ANSIBLE_VERSION}/terraform-provisioner-ansible-linux-amd64_v${TERRAFORM_PROVISIONER_ANSIBLE_VERSION} -O /tmp/.terraform.d/plugins/terraform-provisioner-ansible_v${TERRAFORM_PROVISIONER_ANSIBLE_VERSION} \
+        && chmod +x /tmp/.terraform.d/plugins/terraform-provisioner-ansible_v${TERRAFORM_PROVISIONER_ANSIBLE_VERSION}
+
+COPY --from=build-nss-wrapper /nss_wrapper/obj/src/libnss_wrapper.so /usr/local/lib/libnss_wrapper.so
+COPY summon-gopass /usr/local/bin/summon-gopass
```
Comment (mcanevet) [inline Dockerfile.backup:41]: Please don't commit old version. If we want to recover it we can retrieve it from git history.

**Review by mcanevet — CHANGES_REQUESTED** [review]

**Comment by modridi** [comment]
Btw, this is a draft to test the merge workflow. This is not a PR per se, the code is undone and isn't clean yet.

The comments about the Dockerfile are uesful though.. thanks.

Will close this one and issue a proper PR soon with a cleaner version soon

**Pull request closed without merging**